optaplanner-benchmark/src/test/java/org/optaplanner/benchmark/impl/statistic/StatisticUtilsTest.java

```diff
@@ -0,0 +1,87 @@
+import org.optaplanner.core.api.score.buildin.simplelong.SimpleLongScore;
+
+import static org.junit.Assert.*;
+
+import static org.mockito.Mockito.*;
+
+public class StatisticUtilsTest {
+
+    private static final double DELTA = 0.001d;
+
+    @Test
+    public void testSingleDetermineStandardDeviationDoubles() throws Exception {
+        List<SubSingleBenchmarkResult> subSingleBenchmarkResultList = Arrays.asList(createSubSingleBenchmarkResult(SimpleScore.valueOf(0), 0));
+        assertArrayEquals(new double[]{0d}, StatisticUtils.determineStandardDeviationDoubles(subSingleBenchmarkResultList, SimpleScore.valueOf(0), subSingleBenchmarkResultList.size()), DELTA);
+    }
+
+    @Test
+    public void testMultipleDetermineStandardDeviationDoubles() throws Exception {
+        List<SubSingleBenchmarkResult> subSingleBenchmarkResultList = new ArrayList<SubSingleBenchmarkResult>(2);
+        subSingleBenchmarkResultList.add(createSubSingleBenchmarkResult(SimpleScore.valueOf(-2), 0));
+        subSingleBenchmarkResultList.add(createSubSingleBenchmarkResult(SimpleScore.valueOf(-4), 1));
+        assertArrayEquals(new double[]{1d}, StatisticUtils.determineStandardDeviationDoubles(subSingleBenchmarkResultList, SimpleScore.valueOf(-3), subSingleBenchmarkResultList.size()), DELTA);
+    }
+
+    @Test
+    public void testLargeDetermineStandardDeviationDoubles() throws Exception {
+        long[] subSingleBenchmarkScores = new long[]{-19289560268L, -19345935795L, -19715516752L, -19589259253L, -19390707618L, -19641410518L};
+        List<SubSingleBenchmarkResult> subSingleBenchmarkResultList = new ArrayList<SubSingleBenchmarkResult>(6);
+        SimpleLongScore averageScore = SimpleLongScore.valueOf(0);
+        for (int i = 0; i < subSingleBenchmarkScores.length; i++) {
+            SimpleLongScore current = SimpleLongScore.valueOf(subSingleBenchmarkScores[i]);
+            subSingleBenchmarkResultList.add(createSubSingleBenchmarkResult(current, i));
+            averageScore = averageScore.add(current);
+        }
+        averageScore = averageScore.divide(subSingleBenchmarkScores.length);
+        assertArrayEquals(new double[]{160338212.294d}, StatisticUtils.determineStandardDeviationDoubles(subSingleBenchmarkResultList, averageScore, subSingleBenchmarkResultList.size()), DELTA);
+    }
+
+    @Test
+    public void testGetStandardDeviationString() throws Exception {
+        assertEquals(null, StatisticUtils.getStandardDeviationString(null));
+        assertEquals("2.0", StatisticUtils.getStandardDeviationString(new double[]{2d}));
+        assertEquals("1.41", StatisticUtils.getStandardDeviationString(new double[]{Math.sqrt(2)}));
+        assertEquals("1.60E8", StatisticUtils.getStandardDeviationString(new double[]{160338212.294d}));
+        assertEquals("2.0E9", StatisticUtils.getStandardDeviationString(new double[]{2000000000d}));
+        assertEquals("2.0E10", StatisticUtils.getStandardDeviationString(new double[]{20000000000d}));
```

Comment:
That makes sense. That's it indeed. The other scores are always printed as java literals (so always "." as decimal marker for example, even though nl_BE uses ","). And in the long term we might want everything to be localized to the report locale, but then it should happen for standard deviation and the scores at the same time. We might not want to.

In any case, for now use a new DecimalFormat() instance, fix the locale on Locale.en_US  and supply a pattern so it only prints the "." and the "E" if needed, but no "'" etc. Adjust the comment to indicate that it doesn't use the report locale because the other scores don't use it yet either.